pymongo/cursor.py

```diff
@@ -194,9 +194,7 @@ def __init__(self, collection, filter=None, projection=None, skip=0,
         if allow_disk_use is not None:
             allow_disk_use = validate_boolean("allow_disk_use", allow_disk_use)
 
-        if projection is not None:
-            if not projection:
-                projection = {"_id": 1}
+        if projection not in (None, {}):
```

Comment:
We should remove the {} from this check and go back to `is not None:`. Checking for `{}` is unnecessary.

Reply:
Done.